setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Display a message to the user
+echo "Setting up Verso on Unix-based systems..."
+
+# Function to install packages on Linux and macOS
+install_packages() {
+    if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        echo "Detected Linux. Installing dependencies..."
+        if [ -x "$(command -v apt-get)" ]; then
+            sudo apt-get update
+            sudo apt-get install -y git python3-pip llvm cmake curl
+        elif [ -x "$(command -v yum)" ]; then
```

Comment:
Maybe consider supporting `pacman` as well :)

Reply:
> Maybe consider supporting `pacman` as well :)

I've added support for pacman in next PR do review it.